Displays.3dshapes/CREDITS.md

```diff
@@ -3,23 +3,25 @@
 The credits of 3D models are listed in respective folder.  
 Please fill all 3D models credits and info source data accordingly.  
 
-
 ### Info requirements:
 - Author
 - if the 3D model is script generated, a link to the script generator is required as well as the version and sw used
 - if the 3D model is manually generated, include the source model @ https://github.com/KiCad/packages3D-source
 
-<hr>  
-
-### Displays.3dshapes:  
+<hr>
 
+#### AG12864E
+STEP and WRL models designed by Stefan Olsson
+Author:	Stefan Olsson
+Tool:	- FreeCAD 0.17 Build 11223 64 bit Microsoft windwos platform https://www.freecadweb.org/
 
+### Displays.3dshapes:  
```

Comment:
I'm not sure your merging went as planned:
I think this should be above Line number 13 (#### AG1286)